### PR TITLE
Add dynamic dapp config

### PR DESCRIPTION
**Pull request by iamonuwa**
- [x] Load dapp values from templates
- [x]  Update external links

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    tokenSymbol: '',
+    tokenOwnerWebsite: '',
+    siteUrl: '',
+    totalPresale: '0',
+    referralBP: '0',
+    basisPoint: '0',
+    accountCap: '0'
+  });
+
+  const [favicon, setFavicon] = React.useState<string>('');
```
Comment (altbee) [inline src/pages/index.tsx]: Can you directly import `useState` and use it?

File: src/layout/Head.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { DappMetaData } from 'types';
+
+interface IProps {
+  meta: DappMetaData;
+  favicon: string;
+}
+
+export default ({ meta, favicon }: IProps) => {
```
Comment (altbee) [inline src/layout/Head.tsx]: Can you import the favicon directly?

File: src/pages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+  const history = useHistory();
+
+  React.useEffect(() => {
+    const fetchProjectMeta = async () => {
+      try {
+        let config: DappMetaData = meta;
+        const project: string = history.location.pathname.split('/')[1];
+
+        let images = importAll(
```
Comment (altbee) [inline src/pages/index.tsx]: I think we don't need to use this pattern. I added it as testing purpose

**Review by altbee — APPROVED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (iamonuwa) [reply src/pages/index.tsx:23]: No, it will fail.

File: src/layout/Head.tsx (hunk 2 above)
Comment (iamonuwa) [reply src/layout/Head.tsx]: Favicon is dynamic.

File: src/pages/index.tsx (hunk 3 above)
Comment (iamonuwa) [reply src/pages/index.tsx]: What pattern do we need?

Comment (altbee) [reply src/pages/index.tsx]: I mean we don't need the `importAll` function since we use the ipfs object url

**Pull request merged**